### PR TITLE
Add Caleb Kumar Picture to website

### PR DESCRIPTION
**Pull request by cekcreator**
Add my picture to lab website

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -134,6 +134,11 @@
    email: carson.kohlbrenner@colorado.edu
    pic: carson.jpg
 
+ - name: Caleb Kumar
+   role: Undergrad
```
Comment (alecive) [inline _data/people.yml:138]: This should be lowercase

File: img/people/caleb_kumar.jpg
Comment (alecive) [inline img/people/caleb_kumar.jpg:1]: While this is a great picture, it should be square!

**Review by alecive — CHANGES_REQUESTED** [review]

File: img/people/caleb_kumar.jpg
Comment (cekcreator) [reply img/people/caleb_kumar.jpg:1]: I edited the picture to be as square as possible, let me know if it still does not work. If it does not ill find another one to use!

Comment (alecive) [reply img/people/caleb_kumar.jpg:1]: Unfortunately, it needs to be exactly square. Also, it's too big and it needs to be 400x400 pixels at most. Please coordinate with your supervisor (@calebescobedo ?) , he'll show you how to do it!

File: _data/people.yml (hunk 1 above)
Comment (alecive) [reply _data/people.yml]: We still have problems for some reason:

image

**Review by alecive — CHANGES_REQUESTED** [review]

File: _data/people.yml [hunk 2]
```diff
@@ -134,6 +134,11 @@
    email: carson.kohlbrenner@colorado.edu
    pic: carson.jpg
 
+ - name: Caleb Kumar
+   role: undergrad
+   email: caleb.kumar@colorado.edu
+   pic: caleb_kumar.jpg
```
Comment (alecive) [inline _data/people.yml:140]: ok last thing: this should be the name of your image file! Eitehr change this or the image filename.

**Review by alecive — CHANGES_REQUESTED** [review]

**Comment by alecive** [comment]
I think we are now done! Good job @cekcreator

**Review by alecive — APPROVED** [review]

**Pull request merged**